sdk/digitaltwins/Azure.DigitalTwins.Core/tests/Azure.DigitalTwins.Core.Tests.csproj

```diff
@@ -6,11 +6,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Core.NewtonsoftJson" />
     <PackageReference Include="Microsoft.Extensions.Configuration" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="FluentAssertions" />
+    <PackageReference Update="Newtonsoft.Json" />
```

Comment:
Is this one needed too?

Reply:
it is for annotation, the annotation doesn't come in with the Azure core serialization package.

Reply:
Azure SDK libraries can't take a dependency on Newtonsoft.Json.

Reply:
It's a test. Phew.